packages/jetpack-logo/tests/fixtures.php

```diff
@@ -0,0 +1,20 @@
+<?php
```

Comment:
These are interesting fixtures ;) 

Can we find a way to use what's there in WP? I think it's going to be a huge overhead to define any default WP function like this.